### PR TITLE
Fix genkit init to work with new cli directory

### PR DESCRIPTION
**Pull request by MaesterChestnut**
Also fixes Lint error introduced by recent template change

### CODE REVIEW

**Comment by pavelgj** [comment]
Hmm... just sent #83 but I think your fix is better

File: genkit-tools/cli/config/firebase.index.ts.template [hunk 1]
```diff
@@ -26,7 +26,8 @@ export const menuSuggestionFlow = onFlow(
     }),
   },
   async (subject) => {
-    const prompt = `Suggest an item for the menu of a ${subject} themed restaurant`;
+    const prompt = `Suggest an item for the menu of a
+     ${subject} themed restaurant`;
```
Comment (pavelgj) [inline genkit-tools/cli/config/firebase.index.ts.template:30]: is this intentional?

Comment (MaesterChestnut) [reply genkit-tools/cli/config/firebase.index.ts.template:30]: Yes, but open to a different way of solving the max-len of 80 lint error for Firebase functions

Comment (sam-gc) [reply genkit-tools/cli/config/firebase.index.ts.template:30]: I'd recommend this instead:
```ts
    const prompt =
        `Suggest an item for the menu of a ${subject} themed restaurant`;
```

File: genkit-tools/cli/config/firebase.index.ts.template [hunk 2]
```diff
@@ -26,7 +26,8 @@ export const menuSuggestionFlow = onFlow(
...
 
     const llmResponse = await generate({
       model: $GENKIT_MODEL,
```
Comment (sam-gc) [inline genkit-tools/cli/config/firebase.index.ts.template:33]: This file also needs a newline at the end I believe

**Review by sam-gc — CHANGES_REQUESTED** [review]

**Review by tonybaroneee — APPROVED** [review]

**Review by sam-gc — APPROVED** [review]

**Pull request merged**